CONTRIBUTING.md

```diff
@@ -107,6 +108,15 @@ Enhancement suggestions are tracked as GitHub issues:
 - Changes should be located in the most appropriate directory
 - Please open an issue to suggest a new label.
 
+### Acceptance Tests
+- All submitted changes will go through a round of acceptance tests.
+    - Acceptance includes:
+        - Unit Tests to demonstrate the code works out of context
+        - System Integration tests to demonstrate that the system works as intended
+    - If there are specific or special test scenarios required for the change, they should be documented for any testing
+	- PolySync will perform the acceptance testing periodically in order to get changes incorporated into the source

```

Comment:
Will PS hold off merging to `devel` or `master` until these checks are completed?